src/sqlfluff/dialects/dialect_hive.py

```diff
@@ -448,25 +448,12 @@ class DropStatementSegment(BaseSegment):
     match_grammar = StartsWith("DROP")
     parse_grammar = OneOf(
         Ref("DropDatabaseStatementSegment"),
+        Ref("DropSchemaStatementSegment"),
         Ref("DropTableStatementSegment"),
         # TODO: add other drops
     )
 
 
-@hive_dialect.segment()
-class DropDatabaseStatementSegment(BaseSegment):
-    """A `DROP DATEBASE/SCHEMA` statement."""
-
-    type = "drop_table_statement"
-    match_grammar = Sequence(
-        "DROP",
-        OneOf("DATABASE", "SCHEMA"),
-        Ref("IfExistsGrammar", optional=True),
-        Ref("DatabaseReferenceSegment"),
-        OneOf("RESTRICT", "CASCADE", optional=True),
-    )
```

Comment:
I assume we're inheriting the split out `DropDatabaseStatementSegment` and `DropSchemaStatementSegment` from ANSI so happy with this being removed 👍

Reply:
Yes, these are now inherited from ANSI dialect